package.json

```diff
@@ -5,7 +5,15 @@
   "description": "A starter project for AngularJS",
   "repository": "https://github.com/angular/angular-seed",
   "license": "MIT",
+  "dependencies": {
+    "angular": "^1.5.9",
+    "angular-loader": "^1.5.9",
+    "angular-route": "^1.5.9",
+    "html5-boilerplate": "0.0.1",
+    "shelljs": "^0.7.5"
```

Comment:
Actually, since we copy every frontend asset to `app/lib/`, **nothing** here should be a dependency. Were we to publish an npm package from this project it would have included the built `app/lib/` and no deps.